pkg/controller/dbsyncer/generic_db_to_transport_syncer.go

```diff
@@ -23,47 +23,41 @@ type genericDBToTransportSyncer struct {
 	dbTableName         string
 	transport           transport.Transport
 	transportBundleKey  string
-	syncInterval        time.Duration
 	lastUpdateTimestamp *time.Time
 	createObjFunc       bundle.CreateObjectFunction
 	createBundleFunc    bundle.CreateBundleFunction
+	intervalPolicy      syncerIntervalPolicy
 }
 
 func (syncer *genericDBToTransportSyncer) Start(stopChannel <-chan struct{}) error {
 	ctx, cancelContext := context.WithCancel(context.Background())
 	defer cancelContext()
 
-	syncer.init(ctx)
-	ticker := time.NewTicker(syncer.syncInterval)
+	syncer.init()
 
-	for {
-		select {
-		case <-stopChannel:
-			ticker.Stop()
-			cancelContext()
-			syncer.log.Info("stopped syncer", "table", syncer.dbTableName)
+	go syncer.syncBundle(ctx)
```

Comment:
let's discuss it

Reply:
In previous version 'syncBundles' method was called both from 'init' method and from go routine.
This may potentially cause two 'syncBundles' running concurrently. In addition 'syncBundles' go routine was respawn each ticker interval.
In this version, single go routine 'syncBundles' is created and there is no any reason to call 'syncBundles' from 'init'

Reply:
@OlegSternbergIBM tagging to make comment is not missed.

Reply:
done